postgresql_audit/base.py

```diff
@@ -1,16 +1,18 @@
-import os
+import warnings
 from datetime import timedelta
 from weakref import WeakSet
 
 import sqlalchemy as sa
+from jinja2 import Environment, PackageLoader
```

Comment:
I'd like to avoid this. I think PostgreSQL-Audit shouldn't have a strong dependency on Jinja2.